_data/github/readme_csharp.json

```diff
@@ -0,0 +1 @@
#\n\nA C# client to interact with the unofficial VRChat API. Supports all REST calls specified in https://github.com/vrchatapi/specification.\n\n## Disclaimer\n\nThis is the official response of the VRChat Team (from Tupper more specifically) on the usage of the VRChat API.\n\n> Use of the API using applications other than the approved methods (website, VRChat application) are not officially supported. You may use the API for your own application, but keep these guidelines in mind:\n> * We do not provide documentation or support for the API.\n> * Do not make queries to the API more than once per 60 seconds.\n> * Abuse of the API may result in account termination.\n> * Access to API endpoints may break at any given time, with no warning.\n\nAs stated, this documentation was not created with the help of the official VRChat team. Therefore this documentation is not an official documentation of the VRChat API and may not be always up to date with the latest versions. If you find that a page or endpoint is not longer valid please create an issue and tell us so we can fix it.\n\n## Getting Started\n\n```csharp\nusing io.github.vrchatapi.Api;\nusing io.github.vrchatapi.Client;\nusing System;\nusing System.Diagnostics;\nusing System.Threading.Tasks;\n\nnamespace Example\n{\n    class Program\n    {\n        static void Main(string[] args)\n        {\n            MainAsync().Wait();\n            Console.ReadKey();\n        }\n\n        static async Task MainAsync()\n        {\n            // Configure API key authorization: apiKeyCookie\n            Configuration.Default.Username = VRCUsername;\n            Configuration.Default.Password = VRCPassword;\n\n            try\n            {\n                // Calling \"GetCurrentUser\" will log you in.\n                AuthenticationApi authApi = new AuthenticationApi();\n                var user = await authApi.GetCurrentUserAsync();\n                Console.WriteLine($\"Logged in user {user.DisplayName}, Current Avatar {user.CurrentAvatar}\");\n\n                UsersApi userApi = new UsersApi();\n                var getUser = await userApi.GetUserAsync(user.Id);\n                Console.WriteLine($\"Found user {getUser.DisplayName}, joined {getUser.DateJoined}\");\n\n                WorldsApi worldApi = new WorldsApi();\n                var world = await worldApi.GetWorldAsync(\"wrld_ba913a96-fac4-4048-a062-9aa5db092812\");\n                Console.WriteLine($\"Found world {world.Name}, visits: {world.Visits}\");\n            }\n            catch (ApiException e)\n            {\n                Debug.Print(\"Exception when calling API: \" + e.Message);\n                Debug.Print(\"Status Code: \" + e.ErrorCode);\n                Debug.Print(e.StackTrace);\n            }\n        }\n    }\n}\n```\n\n## Contributing\n\nContributions are welcome, but do not add features that should be handled by the OpenAPI specification.\n\nJoin the [Discord server](https://discord.gg/Ge2APMhPfD) to get in touch with us."}
```

Comment:
What do these files represent?

Reply:
The data files are used for displaying information about the repo and the README themselves. I initially tried with making fetching this information completely dynamic, but GitHub has a very strict rate-limit for unauthenticated requests, so it is instead pre-fetched and served from cache, and then would be auto-updated daily with a cron action.

![bild](https://user-images.githubusercontent.com/5008081/130521294-874b84a3-5d54-4ed5-9b3a-8a7827887eda.png)